CONTRIBUTING.md

```diff
@@ -159,6 +159,12 @@ Thank you for contributing to The Ember Times! Let us know if you can be a guest
     # Create a pull request from your repo on GitHub
     ```
 
+1. Note: If you are making a Handlebars code block, you must use `handlebars` instead of `hbs`. For example:
+
+```handlebars
+{{something-cool}}
```

Comment:
I think the following code example may illustrate when to use `handlebars` better?

```suggestion
{{!-- app/templates/application.hbs --}}

<h1>Hello world!</h1>

{{outlet}}
```